lib/chef/resource/windows_firewall_rule.rb

```diff
@@ -195,12 +236,38 @@ def firewall_command(cmdlet_type)
 
           cmd
         end
+
+        # return the current firewall rule settings for icmp_type and protocol are correct
+        # @return [Boolean]
+        def is_set_properly?(icmp_type, protocol)
```

Comment:
nitpick: it's considered bad style in ruby to preface method names with `is_`, `get_`, etc.